numpy/core/src/multiarray/einsum.dispatch.c.src

```diff
@@ -0,0 +1,2071 @@
+/*
```

Comment:
in the _umath_tests example, The main function implemention is put in *.c.src and the dispatch part put in *.dispatch.c but in this case becomes opposite, which makes the dispatch transform process less convenient.(Intrusive modification)

Reply:
no its the same, there's maybe a misunderstand let me explain to you what is going on.